ui/components/institutional/confirm-add-custodian-token/index.scss

```diff
@@ -0,0 +1,10 @@
+.add_custodian_token_confirm {
+  &__token,
+  &__url {
+    font-size: 14px;
+    height: auto;
+    overflow-wrap: break-word;
+    color: var(--color-background-alternative);
+    opacity: 0.8;
+  }
```

Comment:
Font-size an color can be expressed with props to the Text component

Reply:
Its not specified in the file, but Label takes all the same props as Text. cc @garrettbear and @georgewrmarshall -- is this intended? Should we allow Label to be overridden with different variants etc.

Reply:
True, I've updated it to pass those styles as props 👍🏼

Reply:
Commented here https://github.com/MetaMask/metamask-extension/pull/18261#discussion_r1145313230 I would advise using the default Label styles